uxarray/plot/accessor.py

```diff
@@ -309,6 +340,95 @@ def face_area_distribution(
+
+        Parameters
+        ----------
+        periodic_elements : str, optional, default="exclude"
+            Specifies whether to include or exclude periodic elements in the grid.
+            Options are:
+            - "exclude": Exclude periodic elements,
+            - "ignore": Include periodic elements without any corrections
+            - "split": Split periodic elements.
+        backend : str or None, optional
+            Plotting backend to use. One of ['matplotlib', 'bokeh']. Equivalent to running holoviews.extension(backend)
+        engine: str, optional
+            Engine to use for GeoDataFrame construction. One of ['spatialpandas', 'geopandas']
+        **kwargs : dict
+            Additional keyword arguments passed to `hvplot.paths`.
+
+        Returns
+        -------
+        gdf.hvplot.paths : hvplot.paths
+            A paths plot of the LineString geometries in the unstructured grid
+        """
+        import cartopy.crs as ccrs
+
+        uxarray.plot.utils.backend.assign(backend)
+
+        # Set default style parameters
+        if "color" not in kwargs:
+            kwargs["color"] = "blue"
+        if "line_width" not in kwargs and "linewidth" not in kwargs:
+            if backend == "matplotlib":
+                kwargs["linewidth"] = 2
+            else:
+                kwargs["line_width"] = 2
+        if "title" not in kwargs:
+            kwargs["title"] = "LineString Geometries"
+        if "rasterize" not in kwargs:
+            kwargs["rasterize"] = False
+        if "projection" not in kwargs:
+            kwargs["projection"] = ccrs.PlateCarree()
+        if "clabel" not in kwargs:
+            kwargs["clabel"] = "lines"
+        if "crs" not in kwargs:
+            if "projection" in kwargs:
+                central_longitude = kwargs["projection"].proj4_params.get("lon_0", 0.0)
+            else:
+                central_longitude = 0.0
+            kwargs["crs"] = ccrs.PlateCarree(central_longitude=central_longitude)
+
+        # Create LineString geometries from edge_node_connectivity
+        if self._uxgrid.n_edge > 0 and hasattr(self._uxgrid, "edge_node_connectivity"):
+            edge_conn = self._uxgrid.edge_node_connectivity.values
+            node_lon = self._uxgrid.node_lon.values
+            node_lat = self._uxgrid.node_lat.values
+
+            import geopandas as gpd
+            from shapely.geometry import LineString
+
+            geometries = []
+            for edge in edge_conn:
+                start_node, end_node = edge
+                if start_node >= 0 and end_node >= 0:  # Ensure valid indices
+                    line = LineString(
+                        [
+                            (node_lon[start_node], node_lat[start_node]),
+                            (node_lon[end_node], node_lat[end_node]),
+                        ]
+                    )
+                    geometries.append(line)
```

Comment:
The construction of `geometries` in the for-loop looks more like a helper function rather than plotting, and since it is iterative, it could benefit from Numba I believe. What is the execution performance of this for km-scale grids?